src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -103,7 +104,10 @@ def format_historical_data(data: SnubaTSResult, dataset: Any) -> list[TimeSeries
             formatted_data.append(ts_point)
     else:
         for datum in nested_data:
-            ts_point = TimeSeriesPoint(timestamp=datum.get("time"), value=datum.get("count", 0))
+            #########################
+            count = randrange(1, 100)
```

Comment:
There is no historical data locally so I populate this with random numbers. Seer wasn't happy if I did it all the same number